darwin/future/core/client.py

```diff
@@ -123,33 +121,6 @@ class Page(BaseModel):
     detail: PageDetail
 
 
-class Cursor(ABC):
```

Comment:
Yeah, good call, let's keep these out, unless we come up with a use case that is improved by them